packages/edit-site/src/components/style-book/categories.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * categoryDefinition object.
+ *
+ * @typedef {Object} StyleBookCategory
+ *
+ * @property {string} slug     Category identifier.
+ * @property {string} title    Category title/label.
+ * @property {Array?} blocks   Array of block names to include in the category. Used when blocks are not included in the category by default.
+ * @property {Array?} excludes Array of blocks to exclude from the category. Used when blocks are included in the category by default.
```

Comment:
Wondering if I should just cover this all to TS

Reply:
One step at a time? 

It could easily be a follow-up but if you're going to pull the trigger on it, maybe before we get too deep into the weeds iterating on then Style Book?

Reply:
Didn't see your reply before leaving my review which sort of flip-flopped. I think eventually we'll end up doing it. Now, or in a follow-up, I don't think it matters greatly.

Reply:
Agree 💯 

Tomorrow problem. 😄